client/src/produkter/OpprettProdukt.tsx

```diff
@@ -43,7 +35,7 @@ export default function OpprettProdukt() {
       isoCategory: data.isoCategory,
     };
 
-    const response = await fetch(createProductDraftPath(), {
+    const response = await fetch(`${HM_REGISTER_URL()}/admreg/vendor/api/v1/product/registrations/draftWith`, {
```

Comment:
Dette endepunktet finnes ikke: `${HM_REGISTER_URL()}/admreg/admin/api/v1/product/registrations/draftWith`.

Reply:
Ser det er lagt opp til det i back-end, men da må admin velge hvilken leverandør ref "supplierId" i path: 
@Post("/draftWith/supplier/{supplierId}{?isAccessory}{?isSparePart}")

Vi har ingen skjermbilde for dette enda iallefall men kanskje noko å diskutere down the road(?)

Reply:
Ja enig.. Vi får diskutere det senere